cmd/coordinator/run.go

```diff
@@ -10,6 +10,8 @@ import (
 	"log"
 	"os"
 	"strings"
+	"io/ioutil"
+	"context"
```

Comment:
```suggestion
	"context"
	"io/ioutil"
	"log"
	"os"
	"strings"
```
Just a quick `gofmt`, otherwise looks good